gwcli/client.py

```diff
@@ -335,6 +335,18 @@ def ui_command_auth(self, chap=None):
         """
 
         self.logger.debug("CMD: ../hosts/<client_iqn> auth *")
+
+        if not chap:
+            self.logger.error("To set authentication, specify "
+                              "chap=<user>/<password>")
+            return
+
+        if chap == 'nochap' or chap == '':
+            self.logger.error("CHAP must be disabled for all clients at the "
+                              "same time. Run the 'auth nochap' command from "
+                              "the target's node.")
```

Comment:
Nit: the "node" terminology is a bit confusing to me. Perhaps is can be clarified by stating something like "target's node within gwcli"?